doc/source/ray-overview/index.md

```diff
@@ -615,6 +615,113 @@ ray submit cluster.yaml example.py --start
 
 `````
 
+## Debugging and Monitoring Ray Quick Start
+
+You can use built-in observability tools to monitor and debug Ray applications and clusters.
+
+`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Ray Dashboard: Web GUI to monitor and debug Ray
+:animate: fade-in-slide-down
+
+Ray dashboard provides a visual interface that displays real-time system metrics, node-level resource monitoring, job profiling, and task visualizations. The dashboard is designed to help users understand the performance of their Ray applications and identify potential issues.
+
+```{image} https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/Dashboard-overview.png
+:align: center
+```
+
+````{note}
+To get started with ray dashboard install the Ray default installation as follows.
+
+```bash
+pip install "ray[default]"
+```
+````
+
+```{link-button} ../ray-core/ray-dashboard
+:type: ref
+:text: Learn more about Ray Dashboard.
+:classes: btn-outline-primary btn-block
+```
+
+`````
+
+`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Ray State APIs: CLI to access cluster states
+:animate: fade-in-slide-down
+
+Ray state APIs allow users to conveniently access the current state (snapshot) of Ray through CLI or Python SDK.
+
+````{note}
+To get started with ray state API install the Ray default installation as follows.
+
+```bash
+pip install "ray[default]"
+```
+````
+
+Run the following code.
+
+```{code-block} python
+
+    import ray
```

Comment:
Is this testable? Do you need to move to doc_code folder?

Reply:
It is the same code as the one under state API doc. Let me try moving this.